src/libraries/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -60,6 +60,15 @@ public void CurrentManagedThreadId_Idempotent()
             Assert.Equal(Environment.CurrentManagedThreadId, Environment.CurrentManagedThreadId);
         }
 
+        [Fact]
+        public void ProcessId_MatchesExpectedValue()
+        {
+            Assert.Equal(Environment.ProcessId, Environment.ProcessId);
+
+            using RemoteInvokeHandle handle = RemoteExecutor.Invoke(() => Console.WriteLine(Environment.ProcessId), new RemoteInvokeOptions { StartInfo = new ProcessStartInfo { RedirectStandardOutput = true } });
```

Comment:
We should split this into a different test case since it relies on RemoteExecutor which may not be supported on all platforms (due to no process support).  There may already be a PlatformDetection flag for this /cc @safern.

Reply:
We can just condition this based on: RemoteExecutor.IsSupported:

```
https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.RemoteExecutor/src/RemoteExecutor.cs#L72
```

Reply:
About 4 seconds ahead of you ;-)

Reply:
haha literally 😄